WinImageBuilder.psm1

```diff
@@ -635,8 +635,13 @@ function Resize-VHDImage {
         Write-Host "New partition size: $newSizeGB GB"
 
         if ($NewSize -gt $MinSize) {
+            $global:i = 0

```

Comment:
Not really, we have only the option of recursion with a trailing number of tries